libcodechecker/orm_model.py

```diff
@@ -287,6 +266,16 @@ def __init__(self, bug_hash, author, message, created_at):
         self.created_at = created_at
 
 
+class ReviewStatus(Base):
```

Comment:
If we want to support displaying the whole history of bug status changes, can the bug_hash be a primary key? Multiple status changes can happen to the same bug.